lib/mix/lib/mix/tasks/release.ex

```diff
@@ -404,10 +409,12 @@ defmodule Mix.Tasks.Release do
           ebin/
           include/
           priv/
-      releases
+      releases/
         RELEASE_VSN/
+          consolidated/
           elixir
           iex
+          releases.exs
```

Comment:
I'm not sure about the term "releases" (in plural) inside the directory of a specific release. Maybe `runtime_config.exs`?

Reply:
I had `runtime_config.exs` but given the user file is named `config/releases.exs`, I decided to keep the same name as it is less confusing once you know the context.